doc/releases/changelog-dev.md

```diff
@@ -52,6 +52,9 @@
   The code that checks for qubit wise commuting (QWC) got a performance boost that is noticable
   when many commuting paulis of the same type are measured.
 
+* A `Sum` arithmetic class is added that allows users to represent the sum of operators
```

Comment:
```suggestion
* A `Sum` symbolic class is added that allows users to represent the sum of operators
```

I think we started calling them this way?

Reply:
Yes !